modules/rosa-setting-up-cli.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// getting_started_rosa/rosa-setting-up-cli.adoc
+
+
+[id="rosa-setting-up-cli"]
+= Setting up the `rosa` CLI
+
+To set up the `rosa` CLI:
+
+.Procedure
+
+. Download the latest release of `rosa` for your operating system from the link:https://github.com/openshift/rosa/releases/latest[`rosa` releases page on GitHub].
++
+. (Optional) After downloading a release, you can rename the executable file you download to `rosa`, and then add `rosa` to your path.
++
+. After downloading `rosa`, enable bash completion for `rosa`. Bash completion helps to automatically complete commands and suggest options when you press `Tab`. The command generates a bash completion file for `rosa` and sources it to your current shell session.
++
+To configure your bash shell to load `rosa` completions for each session, add the command to your `bashrc` file (`~/.bashrc` or `~/.profile`).
++
+.Syntax
```

Comment:
Removed. Thanks!

Reply:
Yes, because this is not a procedure, it is acting as a reference topic. There is a new module that I just added that provides a workflow/procedure that includes `init`.